app/api/resources/task_comments.py

```diff
@@ -0,0 +1,177 @@
+from flask_jwt_extended import jwt_required, get_jwt_identity
+from http import HTTPStatus
+from app import messages
+from app.api.dao.task import TaskDAO
+from app.api.dao.task_comment import TaskCommentDAO
+from app.api.resources.common import auth_header_parser
+from app.api.dao.task import MentorshipRelationDAO
+from app.api.dao.user import UserDAO
+from app.api.models.task import *
+from app.api.validations.task_comment import (
+    validate_task_comment_request_data,
+    COMMENT_MAX_LENGTH,
+)
+from app.utils.validation_utils import get_length_validation_error_message
+from app.database.models.task import MentorshipRelationModel
+from app.api.email_utils import send_email_task_accepted
+
+
+task_comments_ns = Namespace(
+    "Comments",
+    description="Operations related to " " task comments in a mentorship relation " "between users",
+)
+add_models_to_namespace(task_comments_ns)
+
+DAO = MentorshipRelationDAO()
+userDAO = UserDAO()
+
+
+
+
```

Comment:
you can remove this if this is not being used, please check first before removing :)